apps/vr-tests-react-components/.storybook/main.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');
```

Comment:
cant we use root storybook config that is being used for all v9 ?

Reply:
I just did a straight copy for speed, I'll try the root one and see what blows up

Reply:
Nope, keeping as is, since we actually want the config to be different. The root one contains codesandbox plugin while this one contains griffel plugin